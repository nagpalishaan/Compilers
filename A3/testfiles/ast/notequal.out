Program
  List
    Function
      IdDecl ID="n"
      Opt
      Opt
        List
          DeclareVar
            IdDecl ID="a"
            Opt
          DeclareVar
            IdDecl ID="b"
            Opt
          AssignVar
            IdUse ID="c"
            Not
              IdUse ID="a"
              IdUse ID="b"
