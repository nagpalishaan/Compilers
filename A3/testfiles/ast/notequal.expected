Program
  List
    Function
      IdDecl ID="n"
      ParamList
        List
      StmtBlock
        List
          DeclareVar
            IdDecl ID="a"
            Opt
          DeclareVar
            IdDecl ID="b"
            Opt
          AssignVar
            IdUse ID="c"
            Not
              IdUse ID="a"
              IdUse ID="b"
